package.json

```diff
@@ -2,20 +2,40 @@
   "name": "@cheqd/mcp",
   "version": "1.0.0",
   "description": "Package with all the mcp toolkits for cheqd",
-  "workspaces": [
-    "packages/*"
-  ],
   "private": "true",
```

Comment:
This directive means the package is not published on NPM at this level. Which is a valid choice (e.g., `creds-libs` is not published at the top level). I just don't understand the point of the current sub-packages, especially the `mcp` one.

Reply:
renamed the subpackage to mcp-toolkit-server